### PR TITLE
refactor: remove registry, offer hooks, send invites

### PR DESCRIPTION
**Pull request by katelynsills**
The corresponding changes for https://github.com/Agoric/agoric-sdk/pull/1256

### CODE REVIEW

File: ui/src/store/operations.js [hunk 1]
```diff
@@ -49,26 +59,24 @@ export function recentOrders(state, orders) {
...
-    // Format is:
-    //   hooks[targetName][hookName] = [hookMethod, ...hookArgs].
-    // Then is called within the wallet as:
-    //   E(target)[hookMethod](...hookArgs)
-    hooks: {
-      publicAPI: {
-        getInvite: ['makeInvite'], // E(publicAPI).makeInvite()
-      },
-    },
+    // TODO: get this from the invite instead in the wallet. We
```
Comment (michaelfig) [inline ui/src/store/operations.js:76]: Not strictly necessary.  The UI is already trusting the dapp backend.

**Review by michaelfig — APPROVED** [review]
LGTM!

File: ui/src/store/operations.js (hunk 1 above)
Comment (katelynsills) [reply ui/src/store/operations.js:76]: Oh, you mean the wallet is trusting the backend? I think we can do better than that, though. If the wallet extracts the instanceHandle and installationHandle from the invite, then the wallet will be trusting Zoe instead

Comment (michaelfig) [reply ui/src/store/operations.js:76]: This code is in the dapp ui, so I meant the ui is trusting the backend.

Comment (katelynsills) [reply ui/src/store/operations.js:76]: Oh, yes, the UI is trusting the dapp backend. I think what I'm talking about here is that the wallet currently takes the instanceHandle and installationHandle unquestioningly from the dapp, but it shouldn't

**Pull request merged**